crimsobot/cogs/cringo.py

```diff
@@ -47,8 +48,8 @@ async def mini(self, ctx: commands.Context) -> None:
 
     async def cringo_main(self, ctx: CrimsoContext, card_size: int = 4) -> None:
         # generate game intro embed
-        name_prefix = CRINGO_RULES['name'][card_size]
-        name_prefix = 'None' if not name_prefix and random.random() > 0.98 else name_prefix
+        name_prefix = CRINGO_RULES['name'][card_size] or 'None'
+        game_is_cursed = not CRINGO_RULES['name'][card_size] and random.random() > 0.98
```

Comment:
```suggestion
        name_prefix = CRINGO_RULES['name'][card_size]
        game_is_cursed = not name_prefix and random.random() > 0.98
        if game_is_cursed:
            name_prefix = 'None'
```
i think we ended up back in `NoneCRINGO` bug here... does this make sense?